### PR TITLE
53 news fragment

### LINKED ISSUE #53: Add a "what's new" fragment to know what friends do

### PR DESCRIPTION
**Pull request by clementcharmillot**
Adds a new fragment `NewsFragment` that allows to add news to it by calling `addNews()`.

The view is scrollable when elements can't fit in the view.

closes #53

### CODE REVIEW

File: app/src/main/res/layout/fragment_news.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        android:layout_marginTop="10dp"
+        android:layout_marginEnd="10dp"
+        android:layout_marginBottom="10dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <LinearLayout
+			android:id="@+id/LL_News"
```
Comment (CristianSafta) [inline app/src/main/res/layout/fragment_news.xml:21]: Perhaps remove a tab so the indentation is consistent

File: app/src/main/java/com/epfl/drawyourpath/misc/NewsFragment.kt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+        // TODO: Create an actual news feed
+    }
+
+    fun addNews(news: News) {
+        parentFragmentManager.beginTransaction()
+            .add(R.id.LL_News, NewsFragmentElement(news.title, news. description, news.action1, news.action2))
+            .commit()
+    }
+}
```
Comment (CristianSafta) [inline app/src/main/java/com/epfl/drawyourpath/misc/NewsFragment.kt:30]: Perhaps add a small comment to explain what the function does and what parameters it takes, for easy lookup

**Review by CristianSafta — CHANGES_REQUESTED** [review]
Overall, the code looks well-structured and clean. Good job! Just a few small remarks.

File: app/src/main/res/layout/fragment_news.xml (hunk 1 above)
Comment (clementcharmillot) [reply app/src/main/res/layout/fragment_news.xml]: You're right, there is a tab instead of the usual 4 spaces. Nice catch.

**Review by CristianSafta — APPROVED** [review]
Thanks for implementing the changes

**Pull request merged**